testutils/src/main/java/com/microsoft/identity/internal/testutils/MockHttpClient.java

```diff
@@ -48,7 +53,19 @@
      */
     private static final Map<HttpRequestMatcher, HttpRequestInterceptor> interceptors = new HashMap<>();
 
+    private static final AtomicBoolean sSaveRequests = new AtomicBoolean(false);
 
+    private static final AtomicLong sTotalRequests = new AtomicLong(0);
```

Comment:
Should these be member variables instead? At least they are being set per instance

Reply:
They're being set in a static method, so... no.  This whole class more-or-less works based on it not being used in parallel across multiple tests.

Reply:
I could move the setters out to the static "factory" methods...